quay-enterprise/security-scanning.md

```diff
@@ -0,0 +1,59 @@
+# Quay Security Scanner
+
+Quay Enterprise supports scanning of container images for known vulnerabilities via a setup security scanner engine such as [Clair](clair.md).
```

Comment:
`Quay Enterprise supports scanning container images for known vulnerabilities with a scanning engine such as [Clair](clair.md). This document explains how to configure Clair with Quay Enterprise.`

(Reference links are preferred in new documents, but I can clean these inline links up later.)